WooCommerce/Classes/Model/BetaFeature.swift

```diff
@@ -48,6 +55,8 @@ extension BetaFeature {
         switch self {
         case .inAppPurchases:
             return ServiceLocator.featureFlagService.isFeatureFlagEnabled(.inAppPurchasesDebugMenu)
+        case .swiftUIPaymentsMenu:
```

Comment:
Does this need to be a beta feature? Are we going to release the SwiftUI Payments Menu once it's finished, or are we keeping it as beta feature? Having two conditions for this feature (flag, beta feature) might be cumbersome to maintain

Reply:
It's a convenience thing for testing and comparing the two implementations – I don't anticipate it being long lived or something we have to maintain. I've added #11111 to remove it when we're done with the rewrite.